verigreen-collector-impl/src/main/java/com/verigreen/collector/common/EmailSender.java

```diff
@@ -140,6 +150,43 @@ public void notifyCommiter(
         if(subject.equals("Verigreen Status - Failure"))
         {
         	status.append(String.format("<tr><td>Temporary branch:</td><td>%s</td></tr></table>", tempBranch));
+        	if(offlineMode.equals("false"))
```

Comment:
@borzamircea - I suggest the following:
1. Manage the URLs in a text file (ini, xml) and use generic tokens (GIT_FAILURE=q-i-got-a-git_failure-status-what-does-it-mean). This way, if we change the links, the logic remains the same (and we need to change the links in a text file and not in the code).
2. From code perspective, please do only string link = String.format("") inside the if's and in the end call one status.append(link) command at the bottom of the if (and else)